CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ### Features
 
+* `[jest-config]` According to #5828, moves `deprecationEntries` from
+  `jest-config` to `jest-validate`.
+  ([#6067](https://github.com/facebook/jest/pull/6067))
+* `[jest-validate]` According to #5828, moves `deprecationEntries` from
+  `jest-config` to `jest-validate`.
```

Comment:
@rickhanlonii Well, there are mentions to two packages. As I wasn't sure if this would be used to an auto-generated doc, I duplicated just like ```
[expect] Add stack trace for async errors (#6008)
[jest-jasmine2] Add stack trace for timeouts (#6008)
``` few lines below

Reply:
Ah, I should read closer, thanks!

Reply:
No worries, and thanks for checking it out!